commons/src/main/java/org/georchestra/commons/configuration/GeorchestraConfiguration.java

```diff
@@ -66,22 +67,15 @@ public GeorchestraConfiguration(String context) {
                 return;
             }
             // loads the application context property file
-            FileInputStream propsFis = null;
             try {
-                try {
-                    // application-context
-                    propsFis = new FileInputStream(new File(contextDataDir, context + ".properties"));
-                    InputStreamReader isr = new InputStreamReader(propsFis, "UTF8");
-                    applicationSpecificProperties.load(isr);
-                } finally {
-                    if (propsFis != null) {
-                        propsFis.close();
-                    }
-                }
-            } catch (Exception e) {
-                activated = false;
-                return;
-            }
+                this.applicationSpecificProperties = this.loadCustomPropertiesFile(context);
+            } catch (Exception e) {}
+
+            // loads common context property file
+            try {
+                this.commonProperties = this.loadPropertiesFile(new File(String.format("%s%s%s",
+                        globalDatadir, File.separator, "common.properties")));
```

Comment:
shared.properties ?

Reply:
wont this confuse/bring bad memories about shared.properties in the template build ? :)

Reply:
You're right, but we want to get rid of the config jar ...

Reply:
In addition, you're free not to use this shared.properties file

Reply:
What about `default.properties` ? This name explicitly explain priority between webapp properties and common/shared/default one

Reply:
> What about default.properties ?

Done with 0ebecfd